docs/en-us/what-is-nacos.md

```diff
@@ -40,7 +40,7 @@ Key features of Nacos:
 
     Centralized management of configuration makes it more convenient for you to achieve stateless services and elastic expansion of service instances on-demand.
 
-    Nacos provides an [easy-to-use UI TODO](xx) to help you manage all of your application or services's configurations. It provides some out-of-box features including configuration version tracking, canary/beta release, configuration rollback, and client configuration update status tracking to ensure the safety and control the risk of configuration change. 
+    Nacos provides an [easy-to-use UI](http://console.nacos.io/nacos/index.html) to help you manage all of your application or services's configurations. It provides some out-of-box features including configuration version tracking, canary/beta release, configuration rollback, and client configuration update status tracking to ensure the safety and control the risk of configuration change. 
```

Comment:
Should we add an DEMO word?

like `Nacos provides an easy-to-use UI ([DEMO](http://console.nacos.io/nacos/index.html))`

Reply:
![1](https://user-images.githubusercontent.com/3291404/95971592-ff138880-0e43-11eb-99d8-8cb4b8784e7c.png)

![2](https://user-images.githubusercontent.com/3291404/95971603-033fa600-0e44-11eb-8d18-b2512caaeb31.png)

which one is better?

Reply:
I'm afraid that the second one will mislead readers into thinking that this demo is the UI itself.

Just for showing way, the second  one is more concise

Reply:
> Should we add an DEMO word?
> 
> like `Nacos provides an easy-to-use UI ([DEMO](http://console.nacos.io/nacos/index.html))`

I agree with this

Reply:
ok, I have modified it